app/models/solidus_paypal_commerce_platform/payment_method.rb

```diff
@@ -75,8 +76,15 @@ def javascript_sdk_url(order: nil, currency: nil)
 
       parameters[:shipping_preference] = 'NO_SHIPPING' if step_names.exclude? 'delivery'
       parameters['enable-funding'] = 'venmo' if options[:enable_venmo]
+      parameters['buyer-country'] = options[:force_buyer_country] if add_buyer_country_to_url?
 
       "https://www.paypal.com/sdk/js?#{parameters.to_query}"
     end
+
+    private
+
+    def add_buyer_country_to_url?
+      options[:force_buyer_country].present? && !Rails.env.production?
```

Comment:
This doesn't give any feedback to the admin user. What do you think to reject the setter when the env is production? Something like:

```
def preferred_force_buyer_country
  return nil if Rails.env.production?
  
  super
end
```

We can also consider removing that preference when the env is production:

```
preference(:force_buyer_country, :string) unless Rails.env.production?
```

Reply:
I think `preference(:force_buyer_country, :string) unless Rails.env.production?` could be a nice edition. Then the admin would never even see this preference in `production`.

The only thing is the admin still can create the preference in the console by: `preferences.update` or `options[:force_buyer_country]`. This would mean we would need to then check the environment again or do something like `respond_to?(:preferred_force_buyer_country) && preferred_force_buyer_country.present?`